neuralprophet/configure.py

```diff
@@ -426,3 +426,12 @@ def init_holidays(self, df=None):
 
 
 ConfigCountryHolidays = Holidays
+
+
+@dataclass
+class Conformal:
+    method: str
+    q_hats: list
+
+
+ConfigConformalPrediction = Conformal
```

Comment:
There is no point in having `Conformal` and `ConfigConformalPrediction` right? For other dataclasses we had this separation because they might contain multiple items, yet this is not the case here.

Reply:
Not necessarily for `ConfigConformalPrediction`, but @ourownstory wants me to put `q_hats` and `method` into a conformal/uncertainty config dataclass instead of being a primary class variable for `NeuralProphet`. So I did.